README.md

```diff
@@ -1,60 +1,137 @@
+    <a href="https://docs.apostrophecms.org/"><strong>Documentation »</strong></a>
     <br />
     <br />
     <a href="http://demo.apostrophecms.com">Demo</a>
     ·
-    <a href="https://portal.productboard.com/apostrophecms/1-product-roadmap/tabs/2-planned">Roadmap</a>
+    <a href="https://productlane.com/edit-roadmap">Roadmap</a>
     ·
     <a href="https://github.com/apostrophecms/apostrophe/issues/new?assignees=&labels=bug,3.0&template=bug_report.md&title=">Report Bug</a>
   </p>
-</p>
+</div>
+
+## About
+
+ApostropheCMS is a full-stack content management system built with Node.js and MongoDB. Content creators can edit directly on live pages without switching between admin interfaces, while developers can build with modern JavaScript or use it headlessly with any frontend framework.
+
+### Key Features
+
+- **🎯 In-Context Editing** - Content creators edit directly on the live page, seeing changes instantly
+- **⚡ Headless-Ready** - Use any frontend framework while keeping the powerful admin experience
+- **🛠️ Developer-First** - Built with Node.js and MongoDB for full-stack JavaScript development
+- **📈 Scales Beautifully** - From small sites to enterprise applications handling millions of pages
+- **🔐 Enterprise Features** - Advanced permissions, workflow management, automated translations, and more
+
+## System Requirements
+
+| Requirement | Version | Installation Notes |
+|-------------|---------|-------------------|
+| **Node.js** | 20.x+ | Use [NVM](https://github.com/nvm-sh/nvm) for version management |
+| **MongoDB** | 6.0+ | [MongoDB Atlas](https://www.mongodb.com/atlas) (cloud) or local install |
+| **npm** | 10.x+ | Included with Node.js |
+
+See our [setup guides](https://docs.apostrophecms.org/guide/development-setup.html) for installation instructions.
+
+## Quick Start
+
+Get ApostropheCMS running locally in minutes:
 
-## About ApostropheCMS
+```bash
+# Option 1: Install CLI globally (recommended for multiple projects)
+npm install -g @apostrophecms/cli
+apos create my-website
+cd my-website
+npm run dev
 
-ApostropheCMS is content software for everyone in an organization. It helps teams of all sizes create dynamic digital experiences with elegance and efficiency by blending powerful features, developer happiness, and a low learning curve for content creators. Apostrophe has powered websites and web apps for organizations large and small for over a decade.
+# Option 2: Use npx for one-time project creation
+npx @apostrophecms/cli create my-website
+cd my-website
+npm run dev
+```
 
-#### Built With
+Your new ApostropheCMS site will be available at `http://localhost:3000` with a powerful admin interface at `/login`.
 
-* [Node](https://nodejs.org/en/)
-* [MongoDB](https://www.mongodb.com/)
-* [Nunjucks](https://mozilla.github.io/nunjucks/)
+### Prefer to Go Headless?
 
-## Getting Started
+ApostropheCMS provides powerful REST APIs for any frontend framework:
```

Comment:
REST API based projects and Astro are absolutely both "headless."

But, this sentence about REST APIs doesn't really work as an intro to both.

Astro is the easiest "headless" starting point. So I would list the Astro starter kit projects in the first subsection of "headless," with a sentence or two expressing why Astro is a great way to get started with a headless build, followed by a separate heading re: REST APIs for a more universal approach.

I would also distinguish the Astro starter kits better. They both are headless sites with visual editing. One is a little more maximal.